src/models/Project.ts

```diff
@@ -666,6 +667,28 @@ class Project {
             .compact()
             .join("-");
     }
+
+    static async clone(api: D2Api, config: Config, id: Id): Promise<Project> {
+        const projectToClone = await Project.get(api, config, id);
+        const clonedDocuments = await promiseMap(projectToClone.documents, async document => {
```

Comment:
Documents kept on the clone appear as REMOVED+ADD. The user may be a little confused, but probably it's not worth doing anything.

Reply:
First time validating a development for this project, but I was confused too about this. If it is an easy fix, @eperedo, I would go ahead with it